pkg/cluster/install.go

```diff
@@ -44,13 +43,33 @@ func (m *manager) AdminUpgrade(ctx context.Context) error {
 	return m.runSteps(ctx, steps)
 }
 
+func (m *manager) Update(ctx context.Context) error {
+	// no-op
+	return nil
+}
+
```

Comment:
Rebase errror. It is missing changed from https://github.com/Azure/ARO-RP/pull/1083/files
```
func (m *manager) Update(ctx context.Context) error {
	var err error
	timeoutCtx, cancel := context.WithTimeout(ctx, 10*time.Minute)
	defer cancel()
	_ = wait.PollImmediateUntil(10*time.Second, func() (bool, error) {
		err = m.ocDynamicValidator.Dynamic(ctx)
		if azureerrors.HasAuthorizationFailedError(err) ||
			azureerrors.HasLinkedAuthorizationFailedError(err) {
			m.log.Print(err)
			return false, nil
		}
		return err == nil, err
	}, timeoutCtx.Done())
	if err != nil {
		return err
	}

	return nil
}
```